src/transformStateWithClones.js

```diff
@@ -65,7 +65,43 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const resArr = [];
+  const newObj = Object.assign({}, state);
+
+  for (let i = 0; i < transforms.length; i++) {
+    let addArr;
+
+    switch (transforms[i].operation) {
```

Comment:
```suggestion
    switch (transform.operation) {
```
якщо використаєш for of